virtualenv.py

```diff
@@ -700,10 +700,15 @@ def main():
         make_environment_relocatable(home_dir)
         return
 
+    if majver == 2 and options.prompt is not None:
+        prompt = options.prompt.decode('UTF-8')
```

Comment:
here you're assuming that `sys.argv` is a list of UTF-8 bytes string in python2, but that may not always be the case on every platform. I believe you should use `sys.getfilesystemencoding()` instead.

Reply:
and even that would not work for windows python2, as this uses the "narrow" WinAPI functions which use replacement characters `?` for those that don't fit the console's 8-bit codepages.

The module https://github.com/Drekin/win-unicode-console has a `get_unicode_argv `returns Unicode version of `sys.argv` obtained by calling the Unicode WinAPI functions `GetCommandLineW` and `CommandLineToArgvW`